_data/edu.yml

```diff
@@ -1,85 +1,148 @@
+    you and your students to create your own!
+
+- title: Low-spec & Cross-platform
+  para: >-
+    Minetest has low system requirements, is compatible with previous
+    generations of hardware, and allows crossplay from any supported device.
+
+
+projects_map: "//umap.openstreetmap.fr/en/map/minetest-projects_893367?scaleControl=false&amp;miniMap=false&amp;scrollWheelZoom=false&amp;zoomControl=true&amp;editMode=disabled&amp;moreControl=true&amp;searchControl=null&amp;tilelayersControl=null&amp;embedControl=null&amp;datalayersControl=true&amp;onLoadPanel=databrowser&amp;captionBar=false&amp;captionMenus=true"
+
+projects:
+
+- title: Minestory
+  author: académie de Dijon - Julien Crémoux
+  tag: School - Heritage Sites
+  country: fr
+  img: minestory.webp
+  para: >-
+    Since 2019, more than 600 primary school students from 30 classes in 3
+    different countries have replicated heritage sites (like Notre-Dame de
+    Paris, the medieval Louvre castle or the Besançon citadel), placed them on
+    an interactive timeline and gave tours and presentations in the virtual
+    world.
+  url: http://minetest.wp.ac-dijon.fr/minestory-frise-immersive-de-sites-du-patrimoine-architectural/
+
+- title: Minetest Classroom
+  author: University of British Columbia - Paul Pickell
+  tag: University - Digital Twin
+  country: ca
+  img: minetest_classroom.webp
+  para: >-
+    A project funded by the University of British Columbia around the use of
+    Minetest as a learning tool and for the production of digital twins of
+    natural and human built environments. It has been used to organize virtual
+    field trips in the context of Covid 19.
+  url: https://eml.ubc.ca/projects/gamifying-forestry/
+
+- title: Permis de déconstruire
+  author: Praticable
+  tag: School - Artistic Residency
+  country: fr
+  img: permisdedeconstruire.webp
+  para: >-
+    Between November 2021 and June 2022, secondary school students from
+    Aulney-sous-Bois in France have reinvented their neighborhood with Minetest.
+    This project, coordinated by the IRI foundation and the teachers, allowed
+    the students to explore the digital practice of cooperative and contributive
+    construction. The projects created emphasized well-being and the
+    neighborhood's conviviality.
+  url: https://praticable.fr/productions/permis-de-deconstruire
+
+- title: UNEJ
+  author: Institut de Recherche et d'Innovation
+  tag: School - Urban Planning
+  country: fr
+  img: unej.webp
+  para: >-
+    The UNEJ project (*Urbanités Numériques En Jeux*) is a 4 years project where
+    teachers and students discover about urban planning while building concrete
+    proposals for their neighborhood's development with the help of
+    professionals from the sector. It is done in connection with the urban
+    transformations carried out for the 2024 Olympics.
+  url: https://tac93.fr/capacitation/unej
+
+- title: Maubeuge en Cubes
```

Comment:
```suggestion
- title: Maubeuge in Cubes
```

Reply:
Isn't "en" appropriate since this is (at least partially) french?